### PR TITLE
Remove Twitter; use Mastodon

### PR DESCRIPTION
**Pull request by dr0i**
See #506.

- update fontawesome to 6.4.2 (gains mastodon) by copy new css to old css
- add webfonts
- remove old awesome fonts
- refactor "fa" (fontawesome) to "fa-brands" for brands (github and twitter)

### CODE REVIEW

**Comment by dr0i** [comment]
Deployed to http://stage.lobid.org/organisations for functional review @acka47 .

**Comment by acka47** [comment]
Something looks strange:
image

**Review by acka47 — APPROVED** [review]
+1

File: app/views/main.scala.html [hunk 1]
```diff
@@ -80,8 +80,8 @@
 					<a href="http://lobid.org/warranty">@Messages.get("main.footer.warranty")</a> | 
 					<a href="https://www.hbz-nrw.de/impressum">@Messages.get("main.footer.imprint")</a> | 
 					<a href="https://github.com/hbz/lobid/blob/master/conf/Datenschutzerklaerung_lobid.textile">@Messages.get("main.footer.data_protection")</a> | 
-					<a href="https://twitter.com/lobidorg"><i class="fa fa-twitter" aria-hidden="true"></i> Twitter</a>&nbsp; 
-					<a href="https://github.com/hbz/lobid-organisations"><i class="fa fa-github" aria-hidden="true"></i> GitHub</a>&nbsp; 
+					<a href="https://openbiblio.social/@@lobid"><i class="fa-brands fa-mastodon" aria-hidden="true"></i> Mastodon</a>&nbsp;
```
Comment (sroertgen) [inline app/views/main.scala.html:83]: is the double `@@` correct?

**Comment by dr0i** [comment]
Deployed to prod, closed.

**Pull request merged**